spec/beaker/shared/host_manager_spec.rb

```diff
@@ -129,7 +129,7 @@ module Shared
           myhosts = host_handler.run_block_on(hosts, nil, { :run_in_parallel => true }) do |host|
             # kind of hacky workaround to remove logger which contains a singleton method injected by rspec
 
-            host.instance_eval("remove_instance_variable(:@logger)")
+            host.instance_eval("remove_instance_variable(:@logger)", __FILE__, __LINE__)
```

Comment:
I think this could be way simpler:
```suggestion
            host.send(:remove_instance_variable, :@logger)
```

https://github.com/voxpupuli/beaker/pull/1793